tests/e2e/framework/kubernetes.go

```diff
@@ -677,6 +691,17 @@ func (k *KubeInfo) deployIstio() error {
 		log.Errorf("Unable to create namespace %s: %s", k.Namespace, err.Error())
 		return err
 	}
+
+	// Deploy the CNI if enabled
+	if *useCNI {
+		err := k.deployCNI()
+		if err != nil {
+			log.Errorf("Unable to deply Istio CNI")
+			return err
+		}
+		time.Sleep(10 * time.Second)
```

Comment:
Anything better we can do than sleeping? can we check for the pods to be created?

Reply:
agreed, I am not keen on the sleep either - not deterministic - would be preferrable to check for completed deployments.  I believe I had implemented this already in the helm API, but I could be mistaken (IOW this sleep may not be necessary, if it is the proper way to handle this is via waiting after a helm deployment for all pods to enter running state.

Reply:
Will check pod status